src/pages/index.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+
+import Home from 'pages/index.page';
+
+describe('Home', () => {
+  it('renders a heading', () => {
+    render(<Home />);
+
+    const heading = screen.getByRole('heading', {
+      name: /welcome to next\.js!/i,
+    });
```

Comment:
Same comment as above, should we [configure](https://testing-library.com/docs/dom-testing-library/api-configuration/) the `test-id-attribute`? This way we can use it in both the UI and Unit tests!

Reply:
Thanks, added in https://github.com/nimblehq/til-web/pull/17/commits/4d0e0c0f2e5b23d38a6fb4b2ff5d2c212afeb9c3